lib/devise_security_extension/hooks/session_lockable.rb

```diff
@@ -4,11 +4,25 @@ def self.extract_email(env)
   rescue
     nil
   end
+
+  def self.extract_default_scope(env)
+    env['warden'].config.default_scope.safe_constantize
```

Comment:
Ahh, so the result of this method will just be a reference to `User`, am I understanding correctly?